luigi/notifications.py

```diff
@@ -311,10 +304,15 @@ def send_email(subject, message, sender, recipients, image_png=None):
     }
 
     subject = _prefix(subject)
-    if not recipients or recipients == (None,) or _email_disabled():
+    if not recipients or recipients == (None,):
+        return
+
+    if _email_disabled_reason():
+        logger.info("Not sending email to %r because reason: %s",
```

Comment:
The message `Not sending email to test@example.com because reason: Not sending email when running from a tty` is a bit awkward and repetitive. How about `Not sending e-mail to test@example.com: e-mail disabled when running from a tty`?